src/components/DownloadImageDialog/utils.ts

```diff
@@ -0,0 +1,37 @@
+import template from 'lodash/template';
+import { Dictionary, OptionalNavigationResource, OsTypesEnum } from './models';
+
+export const FALLBACK_LOGO_UNKNOWN_DEVICE =
+	'https://dashboard.balena-cloud.com/img/unknown-device.svg';
```

Comment:
This really relies on us not changing the name of that file 😕 
Perhaps we should just add the SVG to ui-shared-components as well?

Reply:
I'm still not sure about this point, I'll add a TODO to verify how we can achieve it without duplicating the image